### PR TITLE
Add reset button

### LINKED ISSUE #1: Reset zoom button
It would be nice to have a button to reset the zoom to the original settings.  This might be useful if you use several non-square window zooms.  Alternatively, you might consider constraining window zooms to 1:1 aspect ratio.

### PR DESCRIPTION
**Pull request by kuromusha**
This PR is to enable #1.
add_reset_button

### CODE REVIEW

File: mandelbrot_chaco.py [hunk 1]
```diff
@@ -49,8 +49,10 @@ def dispatch(self, event, suffix):
 
 
 class MandelbrotPlot(HasStrictTraits):
-    ranges = Array(dtype=Float, value=[-2, 1, -1.5, 1.5])
+    _default_ranges = [-2, 1, -1.5, 1.5]
+    ranges = Array(dtype=Float, value=_default_ranges)
     plot = Property(depends_on='ranges')
+    button = Button("Reset of Panning and Zooming")
```
Comment (timdiller) [inline mandelbrot_chaco.py]: The `Button` trait is convenient, and it works fine in this application.  For anything larger, I would want to see this pattern:
```
class MandelbrotPlot(HasStrictTraits):
    # ...
    reset_zoom = Event
    # ...
    @observe('reset_zoom')
    def reset_zoom(self, event):
        # ...
    traits_view = View(
        # ...
        UItem('reset_zoom', editor=ButtonEditor(label="Reset Zoom")),
    )
```
This decouples the view aspects from the model, which is better for testing, etc.  It's probably too much complexity for this simple case, but in principle, I'd prefer to see an `Event` trait modified by a `ButtonEditor`.

Comment (timdiller) [reply mandelbrot_chaco.py]: Also, the name `button` is not descriptive.  As soon as you add a second button, you'll need to go back and rename `button` everywhere it's used.  As in shown in the sample code in the previous comment, a descriptive name is better.

**Review by timdiller — CHANGES_REQUESTED** [review]

File: mandelbrot_chaco.py (hunk 1 above)
Comment (kuromusha) [reply mandelbrot_chaco.py]: @timdiller 
Thank you for your comment. This is very helpful. I modified the code. https://github.com/kuromusha/mandelbrot-chaco/pull/3/commits/f240dd70b71eb481816172546607addf73a90a2e

**Comment by kuromusha** [comment]
https://github.com/kuromusha/mandelbrot-chaco/pull/3/commits/a99a7699db04605ba1f7bdd9c5788c260bc1ef3d is required because of merging the main branch (#4's change).

**Review by timdiller — APPROVED** [review]
LGTM!

**Pull request merged**